README.md

```diff
@@ -102,40 +105,7 @@ Creating a release is as simple as using the
 When creating a release, only a draft release as well as a pull request are created. You still need to publish the draft
 release and merge the Pull Request.
 
-## Credits
-
-The credits of this service heavily go to @thschue and @yeahservice who originally came up with this idea. :rocket:
-
-## License
-
-Please find more information in the [LICENSE](LICENSE) file.
-
-## Documentation
-
-- [Documentation](#documentation)
-    - [Why?](#why)
-    - [How?](#how)
-        - [Specifying the working directory](#specifying-the-working-directory)
-        - [Event Matching](#event-matching)
-        - [Kubernetes Job](#kubernetes-job)
-        - [Kubernetes Job Environment Variables](#kubernetes-job-environment-variables)
-            - [From Events](#from-events)
-            - [From Kubernetes Secrets](#from-kubernetes-secrets)
-            - [From String Literal](#from-string-literal)
-        - [File Handling](#file-handling)
-        - [Silent mode](#silent-mode)
-        - [Resource quotas](#resource-quotas)
-        - [Poll duration](#poll-duration)
-        - [Job namespace](#job-namespace)
-        - [Job image pull policy](#job-image-pull-policy)
-        - [Send start/finished event if the job config.yaml can't be found](#send-startfinished-event-if-the-job-configyaml-cant-be-found)
-        - [Additional Event Data](#additional-event-data)
-        - [Remote Control Plane](#remote-control-plane)
-        - [Job clean-up](#job-clean-up)
-    - [How to validate a job configuration](#how-to-validate-a-job-configuration)
-    - [Endless Possibilities](#endless-possibilities)
-
-## Why?
+## Motivation
```

Comment:
Personally, I liked the Motivation being placed higher up in the `README` because you could see what the service was for at first glance. 

Let me know how you feel about that.

Reply:
fully agree, will move it up